spaceship.zsh

```diff
@@ -642,6 +642,8 @@ spaceship_ruby() {
     ruby_version=$(chruby | sed -n -e 's/ \* //p')
   elif _exists rbenv; then
     ruby_version=$(rbenv version-name)
+  elif _exists asdf; then
```

Comment:
This won't check whether `asdf ruby` plugin is installed. There are several cases where incorrect version might be displayed
- User use `asdf` to manage other languages it'll result in error like`No such plugin`. 
- User has `asdf ruby` installed and no versions are installed then we'll have `version is not installed for ruby`.
- No version is manually configured using either `asdf global ruby` or `asdf local ruby`, Then it'll show `No version set for ruby`. 

Is there a better way to get version using `asdf` for ruby and display version only if `asdf ruby` is installed ?